tests/test_fstring.ml

```diff
@@ -0,0 +1,32 @@
+(*
+ * Module to test FString
+ * DURAND-MARAIS © 2019
+ *)
+
+open Alcotest
+open Core.FString
+
+(* Test center *)
+let test_center_eq _ = 
+  (check string) "center" 
+  "  Center  " (format_string 10 "Center")
+
+(* Test left *)
+let test_left_eq _ =
+  (check string) "left" 
+  "Left  " (format_string ~pos:Left 6 "Left")
+
+(* Test  right *)
+let test_right_eq _ =
+  (check string) "right" 
+  "  Right" (format_string ~pos:Right 7 "Right")
+
+
+(* Test suite for Hello *)
+let suite = [
```

Comment:
Add a test about if the contents size is greater than size argument.

Reply:
Indeed!